www-apps/webdavcgi/webdavcgi-1.1.1.ebuild

```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils toolchain-funcs versionator
+
+DESCRIPTION="A Perl CGI for accessing and sharing files, or calendar/addressbooks via WebDAV."
+HOMEPAGE="http://webdavcgi.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+
+# Provide slotting on minor versions. WebDAV CGI is a web application which
+# can be shared by multiple instances and thus major updates shouldn't be
+# enforced to all users/instances at the same time.
+SLOT="$(get_version_component_range 1-2)"
+
+KEYWORDS="~amd64"
+IUSE="afs git kerberos ldap mysql pdf postgres rcs samba +sqlite +suid"
+
+DEPEND=""
+RDEPEND="afs? ( net-fs/openafs )
+	dev-lang/perl
+	dev-perl/Archive-Zip
+	dev-perl/CGI
+	dev-perl/DateTime
+	dev-perl/DateTime-Format-Human-Duration
+	dev-perl/File-Copy-Link
+	dev-perl/JSON
+	dev-perl/List-MoreUtils
+	dev-perl/MIME-tools
+	dev-perl/PerlIO-gzip
+	dev-perl/Quota
+	dev-perl/TimeDate
+	dev-perl/URI
+	dev-perl/UUID-Tiny
+	dev-perl/XML-Simple
+	git? ( dev-vcs/git )
```

Comment:
Should be correct, according to the [GIT backend documentation](http://webdavcgi.sourceforge.net/doc.html#git), one has to install the git binary.